packages/core/admin/admin/src/pages/UsecasePage/index.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import { useIntl } from 'react-intl';
+import { useHistory } from 'react-router-dom';
+import styled from 'styled-components';
+import { pxToRem, useNotification } from '@strapi/helper-plugin';
+import { Main } from '@strapi/design-system/Main';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Select, Option } from '@strapi/design-system/Select';
+import { TextInput } from '@strapi/design-system/TextInput';
+import { TextButton } from '@strapi/design-system/TextButton';
+import { Button } from '@strapi/design-system/Button';
+import Logo from '../../components/UnauthenticatedLogo';
+import UnauthenticatedLayout, { LayoutContent } from '../../layouts/UnauthenticatedLayout';
+
+export const options = [
+  { label: 'Front-end developer', value: 'front_end_developer' },
+  { label: 'Back-end developer', value: 'back_end_developer' },
+  { label: 'Full-stack developer', value: 'full_stack_developer' },
+  { label: 'Content Manager', value: 'content_manager' },
+  { label: 'Content Creator', value: 'content_creator' },
+  { label: 'Other', value: 'other' },
+];
+
+const TypographyCenter = styled(Typography)`
+  text-align: center;
+`;
+
+const UsecasePage = () => {
+  const toggleNotification = useNotification();
+  const { push, location } = useHistory();
+  const { formatMessage } = useIntl();
+  const [persona, setPersona] = useState();
+  const [personaOther, setPersonaOther] = useState('');
+
+  const isOther = persona === 'other';
+  const isComingFromRegister = location.state?.fromRegister;
+
+  useEffect(() => {
+    if (!isComingFromRegister) {
+      push('/');
+    }
+  }, [isComingFromRegister, push]);
+
+  const handleSubmit = skipPersona => {
+    try {
+      const {
+        state: { email, firstAdmin, firstname },
+      } = location;
+
+      axios({
+        method: 'POST',
+        url: 'https://analytics.strapi.io/register',
+        data: {
+          email,
+          username: firstname,
+          firstAdmin,
+          persona: skipPersona ? undefined : persona,
+          personaOther: skipPersona ? undefined : personaOther,
+        },
+      });
+
+      toggleNotification({
+        type: 'success',
+        message: {
+          id: 'Usecase.notification.success.project-created',
+          defaultMessage: 'Project has been successfully created',
+        },
+      });
+      push('/');
+    } catch (err) {
+      // Silent
+    }
+  };
+
+  if (!isComingFromRegister) {
+    return null;
+  }
```

Comment:
I don't quite understand this case. When would that happen and why would someone like to see an empty page?

Reply:
I added this logic about isComingFromRegister to prevent user to be able to access this page from the URL for example. That's why I redirect the user to the homepage. I guess I could remove the return null part, I added it because the user could have a flash of the usecase page before being redirected, WDYT?

Reply:
I see. I guess I'd prefer to remove it then, because an empty page isn't a nice UX and for whatever reason this might happen :)

Reply:
Yes you should use a `<Redirect />` component in this case.

Reply:
I used the Loading component from helper plugin but Redirect seems cleaner indeed, will do, thank you!